mscore/notetweakerdialog.h

```diff
@@ -0,0 +1,46 @@
+#ifndef NOTETWEAKERDIALOG_H
+#define NOTETWEAKERDIALOG_H
+
+#include <QDialog>
+
+namespace Ui {
+class NoteTweakerDialog;
```

Comment:
Please do `#include "ui_notetweakerdialog.h"` here and omit the one in `notetweakerdialog.cpp`.